### ISSUE TITLE
Update for VS2015

### ISSUE THREAD

**Issue by kipusoep** [warrenbuckley/Umbraco-VS-Project-Items#2]
Is there an update planned for VS2015?

**Comment by warrenbuckley** [comment]
Hi @kipusoep I didn't think this was used by anyone. 
If there is a demand then I can look into it.

**Comment by kipusoep** [comment]
I guess it's just adding it to this line, right? https://github.com/warrenbuckley/Umbraco-VS-Project-Items/blob/master/Umbraco.VS.ProjectItems/Umbraco.VS.ProjectItems/source.extension.vsixmanifest#L12

**Comment by warrenbuckley** [comment]
Yes probably @kipusoep but I know that the VS team has tried to made it easier to build VS extensions in VS2015 so it may have changed.

I am not back hacking & at my laptop truly until a week or so, then I will play catch up & take a look into this.
